inc/api/namespace.php

```diff
@@ -142,6 +142,36 @@ function init_root() {
 
 	// Register Directory endpoints
 	( new Endpoints\Controller\Directory() )->register_routes();
+
+	( new Endpoints\Controller\HealthCheck() )->register_routes();
+
+	add_filter(
+		'rest_endpoints', function ( $endpoints ) {
+			foreach ( $endpoints as $route => $endpoint ) {
+				if ( ! str_contains( $route, 'wp/v2/users' ) ) {
+					continue;
+				}
+
+				foreach ( $endpoint as $index => $handler ) {
+					if ( ! is_array( $handler ) ) {
+						continue;
+					}
+
+					if ( ! isset( $handler['methods'] ) ) {
+						continue;
+					}
+
+					if ( $handler['methods'] !== 'GET' ) {
+						continue;
+					}
```

Comment:
We could unify this in a single condition I think, kind of `! is_array( $handler ) | ! isset( $handler['methods'] ) | $handler['methods'] !== 'GET'`

Reply:
I don't really like a single if statement because it's harder to read in my opinion. Happy to change if you have a strong preference for it.

Reply:
Got it! both options work for me. I'll resolve this thread.

Reply:
I prefer too early return / continue than chaining conditions in this case of scenarios